src/test/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueTest.java

```diff
@@ -156,6 +156,7 @@ public void createAppenderWillReturnANewAppenderEachTime() {
 
     @Test
     public void createAppenderWillThrowWhenQueueIsReadOnly() {
+        assumeFalse(OS.isWindows()); // Read only is not supported on windows
```

Comment:
the convention generally used is
```
assumeFalse("Read-only mode is not supported on Windows", OS.isWindows());
```

Reply:
@tgd I bumped CQ in 2.23 bom too so I would be very happy for you to kick off a 23 release once merged and built

Reply:
@tgd seeking your forgiveness rather than permission to merge this, as we would like to cut a 23 release today